rdbms/util/tuples.cpp

```diff
@@ -16,6 +16,9 @@ tuples::tuples(cell elm1, cell elm2)
   this->elements.insert(set<cell, cellComparator> {elm1});
   this->elements.insert(set<cell, cellComparator> {elm1, elm2});  
 }
+int tuples::get_num_of_elements(){
+  return 2 ;
+}
```

Comment:
This is hard coded and it'll be an obstacle later when we try to scale it up.
use ```return this->num_of_elements``` instead.

Reply:
ok

Reply:
fixed